### DISCUSSION TITLE
Does this repository support energy constraint  mentioned in <<Position-based simulation of continuous materials>>

### DISCUSSION THREAD

**Discussion by blacksino** [InteractiveComputerGraphics/PositionBasedDynamics#113] [Q&A] [answered]

**Comment by janbender** [comment, marked as answer]
Yes. The constraints are called FEMTriangleConstraint and FEMTetraConstraint in the PBD library.

**Reply by blacksino** [reply]
Thank you for your excellent work.

**Reply by Kuprin2000** [reply]
> Yes. The constraints are called FEMTriangleConstraint and FEMTetraConstraint in the PBD library.

Thank you!
